experiments/telemetry/api.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+ChromeUtils.import('resource://gre/modules/TelemetryController.jsm')
+ChromeUtils.import('resource://gre/modules/Services.jsm')
+
+const TELEMETRY_PING_TOPIC = 'frecency-update'
+
+var telemetry = class extends ExtensionAPI {
+  getAPI () {
+    return {
+      experiments: {
+        telemetry: {
+          registerEvents () {
+            Services.telemetry.registerEvents(TELEMETRY_EVENT_CATEGORY, {
```

Comment:
Where are these TELEMETRY_EVENT_* consts defined?

Reply:
Oh, this was old code and I forgot to remove that. It's deleted now.